src/gui/addnewtorrentdialog.cpp

```diff
@@ -337,7 +336,8 @@ void AddNewTorrentDialog::setSavePathHistoryLength(const int value)
 
 void AddNewTorrentDialog::loadState()
 {
-    resize(m_storeDialogSize);
+    if (m_storeDialogSize.get().isValid())
```

Comment:
Maybe add `operator->()` to `SettingValue`?

Reply:
That operator has some implementation restrictions (i.e. only for pointer types) so it isn't suitable for our usage (or I don't know how to implement it)...